doc/examples/rendering/traceSetsArnold.gfr

```diff
@@ -0,0 +1,370 @@
+__children["ShaderAssignment_Plane"]["in"].setInput( __children["ShaderAssignment_Wall"]["out"] )
+__children["ShaderAssignment_Plane"]["filter"].setInput( __children["PathFilter"]["out"] )
+__children["ShaderAssignment_Plane"]["shader"].setInput( __children["RaySwitchShader"]["out"] )
+__children["ShaderAssignment_Plane"]["__contextCompatibility"].setValue( False )
+__children["ShaderAssignment_Plane"]["__uiPosition"].setValue( imath.V2f( 7.23167801, -109.468712 ) )
+__children["PathFilter"]["paths"].setValue( IECore.StringVectorData( [ '/world/geo/plane' ] ) )
+__children["PathFilter"]["__uiPosition"].setValue( imath.V2f( 18.0980415, -102.187462 ) )
+__children["StandardSurface"]["parameters"]["base"].setValue( 1.0 )
+__children["StandardSurface"]["parameters"]["diffuse_roughness"].setValue( 1.0 )
+__children["StandardSurface"]["parameters"]["specular"].setValue( 0.0 )
+__children["StandardSurface"]["__uiPosition"].setValue( imath.V2f( -39.750576, -105.868713 ) )
+__children["StandardSurface_SpecRefl"]["parameters"]["base"].setValue( 1.0 )
+__children["StandardSurface_SpecRefl"]["parameters"]["diffuse_roughness"].setValue( 1.0 )
+__children["StandardSurface_SpecRefl"]["parameters"]["specular"].setValue( 0.0 )
+__children["StandardSurface_SpecRefl"]["__uiPosition"].setValue( imath.V2f( -39.509861, -121.635704 ) )
+__children["TraceSet1"]["parameters"]["passthrough"].setInput( __children["StandardSurface_SpecRefl"]["out"] )
+__children["TraceSet1"]["parameters"]["trace_set"].setValue( 'spheres' )
+__children["TraceSet1"]["parameters"]["inclusive"].setValue( False )
+__children["TraceSet1"]["__uiPosition"].setValue( imath.V2f( -24.7410622, -121.635704 ) )
+Gaffer.Metadata.registerValue( __children["Backdrop4"], 'nodeGadget:color', imath.Color3f( 0.347900003, 0.438600004, 0.5 ) )
+__children["Backdrop4"]["title"].setValue( 'Note: Sets for new locations' )
+__children["Backdrop4"]["description"].setValue( 'Many location-creating scene nodes such as Sphere, Plane, Camera, lights, etc have a Sets plug.\n\nYou can add these newly-created locations to one or more sets by inputting a space-separated list of set names into this plug, instead needing separate Set node(s) downstream.' )
+__children["Backdrop4"]["__uiBound"].setValue( imath.Box2f( imath.V2f( -10, -11.9863358 ), imath.V2f( 25.2358131, 7.88689423 ) ) )
+__children["Backdrop4"]["__uiPosition"].setValue( imath.V2f( 31.9273262, -34.8273163 ) )
+Gaffer.Metadata.registerValue( __children["Backdrop5"], 'nodeGadget:color', imath.Color3f( 0.347900003, 0.438600004, 0.5 ) )
+__children["Backdrop5"]["title"].setValue( 'Note: Trace set names' )
+__children["Backdrop5"]["description"].setValue( 'The `render:` prefix is removed from the set name as part of the conversion to a native Arnold trace set at render time.\n\nAs such, when setting the Trace Set plug on the Trace Set node, you do not need the `render:` prefix, so it is just `spheres` in this example.\n' )
```

Comment:
### Backdrop text
The **"render:"** prefix is removed from the set name as part of the conversion to a native Arnold trace set at render time.

As such, when setting the Trace Set plug on the Trace Set node, you do not need the **"render:"** prefix, so it is just **"spheres"** in this example.

Reply:
Fixed inhttps://github.com/GafferHQ/gaffer/commit/2bd1ad71cc222039afa558e3a083ee62684596af